migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator/sql_schema_calculator_flavour/sqlite.rs

```diff
@@ -1,4 +1,13 @@
 use super::SqlSchemaCalculatorFlavour;
 use crate::flavour::SqliteFlavour;
 
-impl SqlSchemaCalculatorFlavour for SqliteFlavour {}
+impl SqlSchemaCalculatorFlavour for SqliteFlavour {
+    fn column_type_for_native_type(
+        &self,
+        _field: &datamodel::walkers::ScalarFieldWalker<'_>,
+        _scalar_type: datamodel::ScalarType,
+        _native_type_instance: &datamodel_connector::NativeTypeInstance,
+    ) -> sql_schema_describer::ColumnType {
+        unreachable!("column_type_for_native_type on SQLite")
```

Comment:
Is this intended to be filled out later?

Reply:
Probably not — if it does we'll fill it in — basically it's an invariant violation if this gets called at this point, so I'd rather fail hard..